chain/chunks/src/lib.rs

```diff
@@ -265,7 +265,7 @@ impl ShardsManager {
                 },
             };
 
-            self.network_adapter.send(NetworkRequests::PartialEncodedChunkRequest {
+            let _ = self.network_adapter.send(NetworkRequests::PartialEncodedChunkRequest {
```

Comment:
can we use here `do_send()` and avoid `let _ = `?

Reply:
just did that right now :)